packages/common/amplify_db_common/lib/src/connect.vm.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Yeah didn't notice a clear standard either so it seems that we'd need to decide for ourselves.  I do not have any preference as long as we're consistent. 

The suggestions above make sense to me.  If we agree on that I will make sure Analytics is consistent. 

Just waiting on @dnys1 for his thoughts then.

Reply:
Yes, I agree with this and it's pretty close to how I think about it. The only differences I typically make are:
- I use `_stub` instead of `_base` although I have no strong preference towards either.
- I use `_io`/`_js` or `_vm`/`_web` typically depending on whether I'm describing behavior that is specific to web (e.g. DOM interaction) or behavior that just requires JS-interop. The difference is not super important now since we only target one platform that supports JS-interop (web). However, when dart2wasm ships next year, this will be a new target which supports JS-interop but not `dart:html`.